servers/rendering/rasterizer_rd/shaders/screen_space_reflection.glsl

```diff
@@ -0,0 +1,260 @@
+/* clang-format off */
+[compute]
+
+#version 450
+
+VERSION_DEFINES
+
+				/* clang-format on */
+
+				layout(local_size_x = 8, local_size_y = 8, local_size_z = 1) in;
```

Comment:
First call to "layout" needs to be above /* clang-format on */ to avoid this mess I think.

Reply:
Indeed, `clang-format` works ok on C-like GLSL, but it doesn't like our `[vertex]` and `[fragment]` custom attributes, so it needs to be off until *after* the first `layout` statement to work ok. It's weird and ideally that's a bug that should be fixed upstream, but until then we need to work it around this way.

Reply:
If there are `#define`s between the `[vertex]`/`[fragment]` and the first `layout`, we still need to include them in the `off` block. Everything until the first layout should have clang-format disabled.